### ISSUE TITLE
Tags for various oidc-authservice releases (gcr.io/arrikto/kubeflow/oidc-authservice:28c59ef) 

### ISSUE THREAD

**Issue by lalithvaka** [arrikto/oidc-authservice#74]
Hi,

We are in the process of upgrading our KF1.0.2 to KF1.3 and it broke our Logout functionality that we have integrated with Ping OIDC. Am trying to get the right code base to do the debugging in our environment. Is there a way to point or pull the Code for 
gcr.io/arrikto/kubeflow/oidc-authservice:28c59ef from this repo?

@yanniszark @kimwnasptd . 
Appreciate any help. Thank you.

**Comment by lalithvaka** [comment]
Figured it. Thank you.

**Comment by subasathees** [comment]
@lalithvaka , Hi Lalith, what was the solution provided for this logout issue.

**Comment by lalithvaka** [comment]
@subasathees , its been a while I had to apply a custom fix and rebuild the image for our internal use. There seems a bunch of updates to this repo since I last visited. If I  were you, I would look [here](https://github.com/arrikto/oidc-authservice/blob/0c4ea9aa5e962c7f946c1acb65cfae3b0f604817/server.go#L468) . I have used one of the config parameters like staticDestination to add our Enterprise specific URL for global logout.
